examples/ct/ct_handlers_test.go

```diff
@@ -25,6 +24,7 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/mock"
 	"golang.org/x/net/context"
+	"crypto/sha256"
```

Comment:
this import move seems spurious, crypto/sha256 normally comes  in the system imports at the top